<launch>
	<!-- Nodes -->
	<node name="camera" pkg="rtabmap_image" type="camera" output="screen">
		<remap from="camera/image" to="image"/>
		<param name="device_id" value="0" type="int"/>
		<param name="image_hz" value="0" type="double"/>
		<param name="image_width" value="640" type="int"/>
		<param name="image_height" value="480" type="int"/>
	</node>
	
	<node name="visual_attention" pkg="rtabmap" type="visual_attention" output="screen"/>
	
	<!-- 
	<node name="image" pkg="rtabmap_image" type="image_view_qt" output="screen">
		<remap from="image" to="image"/>
	</node>
	-->
	<node name="image_motion" pkg="rtabmap_image" type="image_view_qt" output="screen">
		<remap from="image" to="image_motion"/>
	</node>
	
	<node name="image_motion_local" pkg="rtabmap_image" type="image_view_qt" output="screen">
		<remap from="image" to="image_motion_local"/>
	</node>
	<node name="image_local" pkg="rtabmap_image" type="image_view_qt" output="screen">
		<remap from="image" to="image_local"/>
	</node>
	<node name="image_motion_local_polar" pkg="rtabmap_image" type="image_view_qt" output="screen">
		<remap from="image" to="image_motion_local_polar"/>
	</node>
	<node name="image_motion_local_polar_reconstructed" pkg="rtabmap_image" type="image_view_qt" output="screen">
		<remap from="image" to="image_motion_local_polar_reconstructed"/>
	</node>
</launch>
